src/_includes/soft_mod_hubspot_in_page_banner_cta.liquid

```diff
@@ -1,19 +1,19 @@
 {%- if site.lang == 'es' -%}
-  <span class="hs-cta-wrapper hubspot-in-page-banner" id="hs-cta-wrapper-a4f30da8-3062-4c47-a865-a04a70bfa951">
+  <span class="" id="hs-cta-wrapper-a4f30da8-3062-4c47-a865-a04a70bfa951">
     <span class="hs-cta-node hs-cta-a4f30da8-3062-4c47-a865-a04a70bfa951" id="hs-cta-a4f30da8-3062-4c47-a865-a04a70bfa951">
       <!--[if lte IE 8]><div id="hs-cta-ie-element"></div><![endif]-->
       <a href="https://cta-redirect.hubspot.com/cta/redirect/3042464/a4f30da8-3062-4c47-a865-a04a70bfa951" ></a>
     </span>
     <script charset="utf-8" src="https://js.hscta.net/cta/current.js"></script>
-    <script type="text/javascript"> hbspt.cta.load(3042464, 'a4f30da8-3062-4c47-a865-a04a70bfa951', {}); </script>
+    <script type="text/javascript"> hbspt.cta.load(3042464, '', {}); </script>
   </span>
 {%- else -%}
-  <span class="hs-cta-wrapper hubspot-in-page-banner" id="hs-cta-wrapper-39e3f247-bc43-4085-a52b-73a8d0930da9">
+  <span class="hubspot-in-page-banner" id="hs-cta-wrapper-39e3f247-bc43-4085-a52b-73a8d0930da9">
     <span class="hs-cta-node hs-cta-39e3f247-bc43-4085-a52b-73a8d0930da9" id="hs-cta-39e3f247-bc43-4085-a52b-73a8d0930da9">
       <!--[if lte IE 8]><div id="hs-cta-ie-element"></div><![endif]-->
       <a href="https://cta-redirect.hubspot.com/cta/redirect/3042464/39e3f247-bc43-4085-a52b-73a8d0930da9" ></a>
     </span>
     <script charset="utf-8" src="https://js.hscta.net/cta/current.js"></script>
-    <script type="text/javascript"> hbspt.cta.load(3042464, '39e3f247-bc43-4085-a52b-73a8d0930da9', {}); </script>
+    <script type="text/javascript"> hbspt.cta.load(3042464, '', {}); </script>
   </span>
```

Comment:
I'm a bit confused – this file seems to duplicate the new `hubspot_cta.liquid`?

Reply:
This file was supposed to be deleted. It's been taken care of here: [0377ef6](https://github.com/codurance/site/commit/0377ef60b59d4dcd853ad89f2f295660574ba790)